### PR TITLE
Add fullobjects parameter to content GET request.

### PR DESCRIPTION
**Pull request by tisto**

### CODE REVIEW

File: src/plone/restapi/serializer/dxcontent.py [hunk 1]
```diff
@@ -138,8 +139,23 @@ def __call__(self, version=None):
         if batch.links:
             result['batching'] = batch.links
 
-        result['items'] = [
-            getMultiAdapter((brain, self.request), ISerializeToJsonSummary)()
-            for brain in batch
-        ]
+        if 'fullobjects' in self.request.form.keys():
```
[the remark this answers is not in the document]
Comment (jaroel) [reply src/plone/restapi/serializer/dxcontent.py]: .keys is old style python, just using 'in' should be enough.

File: src/plone/restapi/serializer/dxcontent.py [hunk 2]
```diff
@@ -138,8 +139,23 @@ def __call__(self, version=None):
...
+                }).get('items')
+        else:
+            result['items'] = [
+                getMultiAdapter(
+                    (brain, self.request),
+                    ISerializeToJsonSummary
+                )()
+                for brain in batch
+            ]
+
```
Comment (jaroel) [inline src/plone/restapi/serializer/dxcontent.py]: I think that these two codepaths can be merged into one with something like
```python
query = {'path': {
    'query': '/'.join(self.context.getPhysicalPath()),
    'depth': 1
}
if 'fullobjects' in self.request.form:
    query['fullobjects'] = True

# Contains batching from withing the Lazy serializer
result = handler.search(query)
return results
```

File: src/plone/restapi/serializer/dxcontent.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ebrehault) [reply src/plone/restapi/serializer/dxcontent.py]: Supporting `?fullobjects=0` is possible but that's a breaking change (not a big one but still it needs to be noted). I have no any string opinion about this feature (I do not see a bug difference between adding/not adding the param or setting it to 1/0, but why not)

File: src/plone/restapi/serializer/dxcontent.py (hunk 2 above)
Comment (jaroel) [reply src/plone/restapi/serializer/dxcontent.py]: Right, so one path uses ISerializeToJsonSummary and the other uses ISerializeToJson.

File: src/plone/restapi/serializer/dxcontent.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (tisto) [reply src/plone/restapi/serializer/dxcontent.py]: @ebrehault we just figured out that there was a "sharing" attribute on the top level response of a GET. This means merging this PR will be a breaking change anyways. For what it is worth, I get @jaroel's point about why "fullobjects=0" returning fullobjects can confuse devs.

[the remark this answers is not in the document]
Comment (jaroel) [reply src/plone/restapi/serializer/dxcontent.py]: We'll use the following semantics: 
?fullobjects -> True
?fullobjects=1 -> True
?fullobjects=0 -> False

**Pull request merged**